nats/exactly_once.go

```diff
@@ -18,6 +19,7 @@ type exactlyOnceConsumerConfig struct {
 	FilterSubject       string
 	Handler             Handler
 	DeliverPolicy       nats.DeliverPolicy
+	OptStartTime        time.Time `json:"opt_start_time,omitempty"`
```

Comment:
Name convention seems different (if mimicking nats they use Opt at the end, if not do we need Opt in the name at all)